docs/usage-examples/deleteOne.txt

```diff
@@ -0,0 +1,66 @@
+.. _laravel-delete-one-usage:
+
+=================
+Delete a Document
+=================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: delete one, remove, code example
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can delete a document in a collection by retrieving a single document and calling
+the ``delete()`` method on an Eloquent model or a query builder.
+
+Pass a query filter to the ``where()`` method, sort the matching documents, and call the
```

Comment:
Suggestion: Is "query filter" defined somewhere in these docs yet? If so, it might be good to link to that somewhere. (ie: "For more information about query filters, see ... ")